control-plane-operator/controllers/hostedcontrolplane/cno/testdata/zz_fixture_TestReconcileDeployment_No_private_apiserver_connectivity__proxy_apiserver_address_is_set.yaml

```diff
@@ -153,11 +182,51 @@ spec:
         volumeMounts:
         - mountPath: /etc/hosted-kubernetes
           name: hosted-etc-kube
+      - args:
+        - --service-account-namespace
+        - openshift-network-operator
+        - --service-account-name
+        - cluster-network-operator
+        - --token-file
+        - /var/client-token/token
+        - --token-audience
+        - ""
+        - --kubeconfig
+        - /etc/kubernetes/kubeconfig
+        - --oneshot
+        command:
+        - /usr/bin/control-plane-operator
+        - token-minter
+        imagePullPolicy: IfNotPresent
+        name: init-client-token-minter
+        resources:
+          requests:
+            cpu: 10m
+            memory: 100Mi
+        terminationMessagePolicy: FallbackToLogsOnError
+        volumeMounts:
+        - mountPath: /var/client-token
+          name: client-token
+        - mountPath: /etc/kubernetes
+          name: kubeconfig
       - args:
         - -c
-        - |2-
-
+        - |
+          #!/bin/bash
           set -xeuo pipefail
+
+          while true; do
+             if [[ -f /var/run/secrets/kubernetes.io/hosted/token ]]; then
+                break
+             fi
+             echo "Waiting for client token"
+             sleep 2
+          done
+
+
+          echo "openshift-network-operator" > "/var/run/secrets/kubernetes.io/hosted/namespace"
+          cp "/etc/certificate/ca/ca.crt" "/var/run/secrets/kubernetes.io/hosted/ca.crt"
```

Comment:
These steps don't seem necessary if you're going to build a kubeconfig with these, you can simply set the namespace directly in line 247, and point to /etc/certificate/ca/ca.rt  in line 243 (or wherever you mount the ca there)

Reply:
Sorry, meant to add these comments to the cno code, not the testdata yaml. However, they still apply